lib/runtime/transaction_validity/transaction_validity.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package transactionValidity
+
+import (
+	"errors"
+	"github.com/ChainSafe/gossamer/lib/transaction"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+// TransactionValidityError Information on a transaction's validity and, if valid,
+// on how it relates to other transactions. It is a result of the form:
+// Result<ValidTransaction, TransactionValidityError>
+type TransactionValidityError scale.VaryingDataType
```

Comment:
`exported: type name will be used as transactionValidity.TransactionValidityError by other packages, and that stutters; consider calling this Error (revive`
Maybe move this to the `runtime` package if you want to match the name with substrate.  The stutter is kinda ugly.

Reply:
So just get rid of the transaction_validity package and put these files in runtime? So can access like runtime.TransactionValidityError?

Reply:
If I may, maybe we can add a subpackage `lib/runtime/errors` and move that new code there?